### PR TITLE
create blog

### PR DESCRIPTION
**Pull request by MinaZhen**

### CODE REVIEW

File: staff/minazhen/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Blog</title>
+</head>
+
+<body>
+    <header>
+        <div>
```
Comment (lgimeneza) [inline staff/minazhen/blog/index.html]: div is not semantic

File: staff/minazhen/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            <a href="https://es.wikipedia.org/wiki/Logo_(canal_de_televisi%C3%B3n)">
```
Comment (aperacaula) [inline staff/minazhen/blog/index.html]: la <a> no tiene etiqueta de cierre, no?

File: staff/minazhen/blog/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                <li>
+                    <a href="#t4">Título cuatro</a>
+                </li>
+            </ul>
+        </nav>
+    </aside>
+    <main>
+
+        <section>
+            <img src="http://fotos.trucoteca.com/noticias/usuarios/1378001224-minecraft-twitter-header-600x200.jpg" alt="Minecraft">
```
Comment (lgimeneza) [inline staff/minazhen/blog/index.html:65]: in a header?

Comment (MinaZhen) [reply staff/minazhen/blog/index.html:65]: Yes, this is going to be a fixed menu.

File: staff/minazhen/blog/index.html (hunk 1 above)
Comment (MinaZhen) [reply staff/minazhen/blog/index.html]: That's not important now... is just thinking about css

File: staff/minazhen/blog/index.html (hunk 2 above)
Comment (MinaZhen) [reply staff/minazhen/blog/index.html]: touché

File: staff/minazhen/blog/index.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            <h3 id="t3">Título 3</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore consequatur quod quos repudiandae hic est harum.
+                Magnam quasi eaque, beatae dignissimos quia quisquam. Ducimus nisi a illo culpa eveniet atque.</p>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cupiditate, porro. A sed laborum quaerat ab eaque,
+                dolorum dolores, suscipit voluptates eligendi quia corporis autem molestiae quae recusandae nam porro veniam?</p>
+        </section>
+        <section>
+            <img src="https://www.calliaweb.co.uk/wp-content/uploads/2015/10/600x200.jpg" alt="">
```
Comment (agandia9) [inline staff/minazhen/blog/index.html]: empty alt
**caution**

**Pull request merged**